### ISSUE TITLE
`export` statement seems not working

### ISSUE THREAD

**Issue by CicholGricenchos** [touilleMan/godot-python#98]
I am using Godot 3.0.2 with godot-python 0.11.1 downloaded from the AssetLib. I created a Label node and attached the following script to the Label.

```python
from godot import exposed, export
from godot.bindings import *
from godot.globals import *


@exposed
class Hello(Label):

    # member variables here, example:
    a = export(int)
    b = export(str, default='foo')

    def _ready(self):
        print(123)
        """
        Called every time the node is added to the scene.
        Initialization here.
        """
        pass
```

However I cannot see the `a` and `b` property in the editor inspector. I tried restart the engine but it doesn't help. The script was run because I can see the `123` printed.

Is it a godot-python issue? or due to my incorrect usage?

Thanks!

**Comment by touilleMan** [comment]
Hi,

You're right, it seems `export` doesn't work anymore :(
I'm a bit puzzle why is that so given this part of the code (both in Godot-Python and in Godot Pythonscript) should be pretty stable.
I'll investigate...

**Comment by CicholGricenchos** [comment]
Yes. Actually I am working on a godot-ruby binding. And I tried different settings (retrieved from `get_property_list()`) for script manifest but just cannot get the `export` to work. Then I found godot-python has similar problem. So it's likely to be a pluginscript api issue.

**Comment by touilleMan** [comment]
@CicholGricenchos It seems the export feature of pluginscript was not entirely implemented ^^
see https://github.com/godotengine/godot/pull/19585

**Comment by CicholGricenchos** [comment]
@touilleMan Great job! 😄😄😄
I can confirm the issue is gone on master.
